src/common/constants/constants.ts

```diff
@@ -1,4 +1,5 @@
-import { Genre, Role } from './enums'
+import { Genre, Input, Role } from "./enums";
+import { TextFieldProps } from "@mui/material";
```

Comment:
💭 Not a huge fan of importing MUI to the constants folder.  I see why you're doing it, but I wonder if there's a different way.